crypto/kdf/tls1_prf.c

```diff
@@ -184,7 +184,8 @@ static int tls1_prf_P_hash(const EVP_MD *md,
     int ret = 0;
 
     chunk = EVP_MD_size(md);
-    OPENSSL_assert(chunk >= 0);
+    if (!ossl_assert(chunk > 0))
+        goto err;
```

Comment:
Yes, that is correct, this note should be disregarded just like the one above it.